src/main/java/org/cactoos/map/MapOf.java

```diff
@@ -143,17 +140,22 @@ public MapOf(final Map<X, Y> src,
         );
     }
 
+    /**
+     * Ctor.
+     * @param scalar The scalar for Map.
+     */
+    public MapOf(final Scalar<Map<X, Y>> scalar) {
+        this(new Unchecked<>(scalar).value().entrySet());
```

Comment:
@andreoss what is the `Scalar` for? If we want to create a map from an existing one, then let's just take a map in the constructor. You can then use `putAll` to add all the elements of the original map.

Reply:
@victornoel Removed that ctor.